style.css

```diff
@@ -0,0 +1,86 @@
+h1{	
+		font-size: 80px;
+		text-align: center; 
+		color: Yellow;
+		margin-bottom: -10px;
+		text-decoration: underline;
+
+	}
+
+h3{
+		text-align: center; 
+		color: blue;
+	}
+
+h3.flea {
+		text-align:left;
+}	
+
+p {
```

Comment:
@LuisLithgow Just a tip, you could have declared your 3 margin statements in a single line like this:
`margin: 0 20px 50px 30px;`
The numbers, in order, are top, right, bottom, left.